core/src/main/java/com/linecorp/armeria/internal/DefaultTimeoutController.java

```diff
@@ -149,6 +149,13 @@ public void resetTimeout(long newTimeoutMillis) {
         }
     }
 
+    @Override
+    public void timeoutNow() {
+        ensureInitialized();
+        cancelTimeout();
+        timeoutTask.run();
```

Comment:
Does it need a guard here? 
```java
if (cancelTimeout()) {
    timeoutTask.run();
}
```

Reply:
I guess yes, unless there's anything that must be done even after cancellation.